EMIP/1-1000/EMIP00816.xml

```diff
@@ -82,4 +283,4 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <ab/>
```

Comment:
You need to add a 'change' statement with your ID which gives the broad details of what you have done so that if any changes are made later the reason for changes is clear. In this case you can say something like, 'added ms contents, additions, extras,...' and mention each part that you have edited.